src/main/java/stsjorbsmod/powers/ReapAndSowPower.java

```diff
@@ -0,0 +1,29 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.characters.ReapAndSowSaveData;
+
+
+public class ReapAndSowPower extends CustomJorbsModPower {
```

Comment:
I really like the use of the power as a visual indicator! However, consider that there are effects that can remove buffs from the character (particularly, the `Amnesia` curse), and that the player could theoretically stack multiple instances of `ReapAndSow`; I think I would suggest using the power rather than the savedata to track the overkill damage while the combat is still happening and handle the stacking behavior, and only commit the power amount to `ReapAndSowSaveData.reapAndSowDamage` at the end of combat (I think `onVictory` is the hook method you'd want, but you'd want to double check the ordering for that vs save data being saved).